reactive/containerd.py

```diff
@@ -133,6 +133,12 @@ def upgrade_charm():
     config_changed()
 
 
+@hook('pre-series-upgrade')
+def pre_series_upgrade():
+    """Set the status during series upgrade."""
+    status_set('blocked', 'Series upgrade in progress')
```

Comment:
Seems like this status will get overwritten by the charm_status atexit function. I recommend setting this status in the charm_status atexit function or refactoring the charm to use layer-status.

Reply:
Thanks, I missed the atexit.